client/src/main.ts

```diff
@@ -150,8 +150,9 @@ async function setupApp(): Promise<void> {
     window.electronAPI.receive('open-link', async (link: string) => {
       if ((await claimLinkValidator(link)).validity === Validity.Valid) {
         await handleJoinLink(link);
-      } else if ((await fileLinkValidator(link)).validity === Validity.Valid) {
-        await handleFileLink(link, informationManager, t);
+        // Commented until we can handle file links
```

Comment:
```suggestion
        // FIXME: Commented until we can handle file links
```
For that type of comment, I recommend using the prefix `FIXME:` or `TODO:` some extension / IDE highlight those comment and we can easily search for them using `grep`.

If you can also add a link to and issue that fix the link issue, that would be greeter